src/NexusMods.App.UI/Pages/Settings/SettingsPageViewModel.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.ObjectModel;
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using DynamicData;
+using JetBrains.Annotations;
+using NexusMods.Abstractions.Settings;
+using NexusMods.App.UI.Controls.Settings.SettingEntries;
+using NexusMods.App.UI.Windows;
+using NexusMods.App.UI.WorkspaceSystem;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace NexusMods.App.UI.Pages.Settings;
+
+[UsedImplicitly]
+public class SettingsPageViewModel : APageViewModel<ISettingsPageViewModel>, ISettingsPageViewModel
+{
+    public ReactiveCommand<Unit, Unit> SaveCommand { get; }
+    public ReactiveCommand<Unit, Unit> CancelCommand { get; }
```

Comment:
Cancel should reset all CurrentValues to Previous values in the value containers. 
Or alternatively close the page.